packages/react-dom/src/client/inputValueTracking.js

```diff
@@ -63,20 +63,21 @@ function trackValueOnNode(node: any): ?ValueTracker {
   // (needed for certain tests that spyOn input values and Safari)
   if (
     node.hasOwnProperty(valueField) ||
+    typeof descriptor === 'undefined' ||
```

Comment:
In theory this could happen and then the code would crash.
I'm not sure this is possible in practice.

Reply:
Happened to me in practice :)